services/ext/api.go

```diff
@@ -814,6 +814,10 @@ func (api *PublicAPI) MarkAllRead(ctx context.Context, chatID string) error {
 	return api.service.messenger.MarkAllRead(ctx, chatID)
 }
 
+func (api *PublicAPI) MarkCommunityNotificationRead(ctx context.Context, communityID string) error {
```

Comment:
Even if we only need one argument `communityID` today, it is preferable to accept a Request struct because then new keys can be added in a backwards compatible way. This has been a problem a few times in the past that required coordination between releases in desktop/mobile. Example https://github.com/status-im/status-go/blob/c27384680a119eeeefc887e2aa0591e56387f121/protocol/requests/send_contact_request.go#L14

Then you would also be able to follow the convention to define a `Validate()` function and use it to check if the `communityID` argument is present, something currently missing in the PR.